stream-applications-build/pom.xml

```diff
@@ -44,8 +44,9 @@
 		<spring-kafka.version>2.8.2</spring-kafka.version>
 		<spring-rabbit.version>2.4.2</spring-rabbit.version>
 		<spring-cloud-function.version>3.2.1</spring-cloud-function.version>
+		<spring-cloud-starters.version>3.1.0</spring-cloud-starters.version>
 		<spring-data-geode-test.version>0.0.21.RELEASE</spring-data-geode-test.version>
-		<test-containers.version>1.15.2</test-containers.version>
+		<test-containers.version>1.15.3</test-containers.version>
```

Comment:
??? According Maven we have already `1.16.3`: https://mvnrepository.com/artifact/org.testcontainers/testcontainers. 🤷

Reply:
Got it. I will make sure this is at the proper version. I thought that was 1.15.3 but will 2x check what SpringBoot is also recommending in their deps.

Reply:
This is test dependency, so as long as it does not require `main` changes, we are good to go.